tests/test_storage.py

```diff
@@ -586,6 +688,39 @@ def test_delete_single_submission_or_reply_race_guard(homedir, mocker):
     mock_remove.call_count == 1
 
 
+def test_rename_file_does_not_throw(homedir):
+    """
+    If file cannot be found then OSError is caught and logged.
+    """
+    try:
```

Comment:
You should use the `with pytest.raises(OsError)` syntax here.

Reply:
I like your idea but looks like pytest doesn't have anything like `pytest.does_not_raise(OSError)`.

Reply:
Ah wait sorry I got that backwards. You should remove the `try/except` block and just let the `OsError` fail the test. Then it will contain the whole traceback to help debug what went wrong. Unless there is a reason to explicitly fail here and not have the exception cause the failure?

Reply:
Oh yeah, good call. Keeps it simple.